src/PrestaShopBundle/Controller/Admin/Configure/AdvancedParameters/WebserviceController.php

```diff
@@ -66,17 +68,63 @@ public function indexAction(WebserviceKeyFilters $filters, Request $request)
 
         $configurationWarnings = $this->lookForWarnings();
 
+        $webserviceConfiguration = $this->get('prestashop.admin.webservice.form_data_provider')->getData();
+        $webserviceIsEnabled = $webserviceConfiguration['webservice_configuration']['enable_webservice'];
```

Comment:
```suggestion
        $webserviceIsEnabled = (bool) $webserviceConfiguration['webservice_configuration']['enable_webservice'];
```

Reply:
Fixed in https://github.com/PrestaShop/PrestaShop/pull/19335/commits/db6866eec2cf932f050ef74981bd353c4e967f52